.ahub/tcchecker-tca/config.yaml

```diff
@@ -41,3 +41,58 @@ test:
     positiveTestCase:
       - condition:
         - inverse: negativeTestCase
+
+  - name: NN_Compiler
+    testCaseLanguage: CPP
+    testFW: GTEST
+    testCaseFolder:
+      - /compiler/angkor
+      - /compiler/arser
+      - /compiler/circle2circle
+      - /compiler/circle-quantizer
+      - /compiler/crew
+      - /compiler/cwrap
+      - /compiler/foder
+      - /compiler/hermes
+      - /compiler/hermes-std
+      - /compiler/loco
+      - /compiler/locomotiv
+      - /compiler/locop
+      - /compiler/logo
+      - /compiler/logo-core
+      - /compiler/luci
+      - /compiler/luci-interpreter
+      - /compiler/luci-eval-driver
+      - /compiler/luci-pass-value-test
+      - /compiler/luci-value-test
+      - /compiler/mio-circle
+      - /compiler/mio-tflite
+      - /compiler/oops
+      - /compiler/pepper-assert
+      - /compiler/pepper-str
+      - /compiler/pepper-strcast
+      - /compiler/pp
+      - /compiler/record-minmax
+      - /compiler/safemain
+      - /compiler/souschef
+      - /compiler/tflite2circle
```

Comment:
Could you tell me where you got the list? I think some are missing.

Reply:
@seanshpark , from https://github.com/Samsung/ONE/blob/master/compiler/.ahub/tcchecker-tca/config.yaml
If there is anything to be added, please append here or another PR as you prefer. :)

Reply:
> If there is anything to be added, please append here or another PR as you prefer. :)

Thanks! I wasn't aware of this file but there was some changes in modules to release binary.
I'll post in another PR.